contrib/format-excel/README.md

```diff
@@ -0,0 +1,36 @@
+# Excel Format Plugin
+This plugin enables Drill to read Microsoft Excel files.  This format is best used with Excel files that do not have extensive formatting, however it will work with formatted files, by allowing you to define a region within the file where the data is.  
+
+The plugin will automatically evaluate cells which contain formulae.  
+
+## Plugin Configuration 
+This plugin has several configuration variables which must be set in order to read Excel files effectively.  Since Excel files often contain other elements besides data, you can use the configuration variables to define a region within your spreadsheet in which Drill should extract data.  This is potentially useful if your spreadsheet contains a lot of formatting or other complications. 
+
+* `headerRow`:  Set to -1 if there are no column headers.  
+* `lastRow`:  This defines the last row of your data.  The default is an arbitrary large number.  You only will need to set this if you want Drill to stop reading at a specific location.
```

Comment:
I ma not sure that referring as `you ` is common documentation language style...